### ISSUE TITLE
Possibly sunset the mturk experiment code

### ISSUE THREAD

**Issue by misaugstad** [ProjectSidewalk/SidewalkWebpage#1243]
We made a diverging version of this code base that we used specifically for MTurk experiments in Fall 2017. As expected, this version of the code will be very out of date soon, and thus won't really be useful for further experiments. We are going to be working on making the main code base more flexible over the next couple months, so I think that all future experiments should be created using this master branch.

Tasks relating to sunsetting (of which we could choose a subset to do):
1. Close all issues related to this diverging branch
2. Shut down the mturk server, since we don't plan on using it again and we have a database dump
3. Maybe we could create a fork of this repo for that branch, so that we have easier access to it in the future, in case we want to look at the code for reference.

My main thing is all the issues related to that branch. We aren't really going to modify that code anymore, so I'd love to close them all and focus on the production code.

**Comment by jonfroehlich** [comment]
@misaugstad what's the status of this now?

**Comment by misaugstad** [comment]
I think the only thing that has changed is that I created an "Old MTurk" label and applied it to all the issues for that particular branch. It is branch sidewalk-mturk-v2. Haven't done anything else here yet.

**Comment by misaugstad** [comment]
Okay I made a new fork from that branch [here](https://github.com/ProjectSidewalk/SidewalkWebpageMTurk) and archived it. And then I deleted all the extra relevant branches that we were holding onto in this repo. If we even considered using an old branch, we'd probably use it from either the [SidewalkWebpageMTurk](https://github.com/ProjectSidewalk/SidewalkWebpageMTurk) or [SidewalkWebpageDC](https://github.com/ProjectSidewalk/SidewalkWebpageDC) repo anyway, so no harm in deleting them here.

I don't fully remember what happened with the server that was running with the sidewalk-mturk-v2 code, but I have to imagine that it's no longer up and running.

Next I'll close all the Github issues related to the old MTurk code. And that should be everything needed to close out this issue!
